utils/check_copies.py

```diff
@@ -326,24 +378,60 @@ def _find_text_in_file(filename, start_prompt, end_prompt):
 
 def check_model_list_copy(overwrite=False, max_per_line=119):
     """Check the model lists in the README and index.rst are consistent and maybe `overwrite`."""
+
+    # If the introduction or the conclusion of the list change, the prompts may need to be updated.
     rst_list, start_index, end_index, lines = _find_text_in_file(
         filename=os.path.join(PATH_TO_DOCS, "index.rst"),
         start_prompt="    This list is updated automatically from the README",
         end_prompt="Supported frameworks",
     )
-    md_list = get_model_list()
-    converted_list = convert_to_rst(md_list, max_per_line=max_per_line)
+    md_list = get_model_list(
+        filename="README.md",
+        start_prompt="🤗 Transformers currently provides the following architectures",
+        end_prompt="1. Want to contribute a new model?",
+    )
+
+    converted_rst_list = convert_to_rst(md_list, max_per_line=max_per_line)
+
+    converted_md_lists = []
+    for filename, value in LOCALIZED_READMES.items():
+        _start_prompt, _end_prompt, _format_model_list = (
+            value["start_prompt"],
+            value["end_prompt"],
+            value["format_model_list"],
+        )
```

Comment:
With the formatting being like this, it would actually take less space and be more readable by doing three assignments on three different lines